docs/API/metadata.md

```diff
@@ -92,40 +92,50 @@ This defines the dataset resource, and contains metadata received from the
 The `metadata.log` data is represented under the key `dataset.metalog` and can
 be queried as part of the entire dataset using the `dataset` key, as a discrete
 sub-document using `dataset.metalog` in specific "sections" such as
 `dataset.metalog.pbench` or targeting a specific value like
 `dataset.metalog.pbench.script`.
 
+#### `dataset.name`
+
+This namespace includes the resource name, which can be modified by the owner
+of the dataset by setting the metadata key `dataset.name`. All other key values
+in this namespace are controlled by the server and cannot be changed by the
+client.
+
 ### Server namespace
 
 This defines internal Pbench Server management state related to a dataset
 that's not inherent to the representation of the user's performance metrics.
 These are generally not useful to clients, and some can be large. There are
 three values in this namespace that clients can modify:
 
-* `server.deletion` is a date after which the Pbench Server may choose to
-delete the dataset. This is computed when a dataset is received based on user
-profile preferences and server configuration; but it can be modified by the
-owner of the dataset, as long as the new timestamp remains within the maximum
-allowed server data retention period.
-* `server.archiveonly` is a boolean that can be set to a boolean True when a
-dataset is first uploaded to prevent the Pbench Server from unpacking or
-indexing the dataset. That is, the server will archive the dataset and it can
-be retrieved for offline analysis but the server will do nothing else with it.
-The value can be specified as "t", "true", "y" or "yes" (case insensitive) for
-True, and "f", "false", "n", or "no" (also case insensitive) for False. Note
-that this is currently only interpreted by the Pbench Server when a dataset is
-first uploaded, and will inhibit unpacking and indexing the dataset. It can be
+#### `server.deletion`
+
+This is a date after which the Pbench Server may choose to delete the dataset.
+This is computed when a dataset is received based on user profile preferences
+and server configuration; but it can be modified by the owner of the dataset,
+as long as the new timestamp remains within the maximum allowed
+[server data retention period](./V1/server_settings.md#dataset-lifetime).
+
+#### `server.archiveonly`
+
+This is a boolean that can be set to a boolean True when a dataset is first
+uploaded to prevent the Pbench Server from unpacking or indexing the dataset.
+The server will archive the dataset and it can be retrieved for offline
+analysis but the server will do nothing else with it. The value can be
+specified as "t", "true", "y" or "yes" (case insensitive) for True, and "f",
+"false", "n", or "no" (also case insensitive) for False. Note that this is
+currently only interpreted by the Pbench Server when a dataset is first
+uploaded, and will inhibit unpacking and indexing the dataset. It can be
 changed later, but the server currently takes no action on such changes.
```

Comment:
I would drop the "and will inhibit unpacking and indexing the dataset" since you describe that in the first sentence (and it only has an effect if the value is "true").  Also, I recommend phrasing like, "Note that setting this value is only meaningful in the upload request; changing it later will have no effect."